client.go

```diff
@@ -414,11 +420,17 @@ func (c *Client) Deploy(path string) (err error) {
 	}
 
 	// Push the image for the named service to the configured registry
+	if c.verbose {
+		fmt.Println("\nPushing Function image to the registry:")
```

Comment:
Why the ":" at the end of this message (as well as below on 432)?

Reply:
I can remove that, but the output of `docker push/build` follows these lines

Reply:
Oh that makes sense then